### PR TITLE
[ggj] Add overall test coverage

### PR DESCRIPTION
**Pull request by summer-ji-eng**
- Add codecov config file
- Add codecov github action
- Add codecov badge showing on readMe

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -60,6 +60,11 @@ jobs:
           name: artifacts
           path: ~/artifacts
 
+      - name: Tests Coverage
```
Comment (miraleung) [inline .github/workflows/ci.yaml:63]: Nit: "Code Coverage"

File: codecov.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+codecov:
+  notify:
+    require_ci_to_pass: yes
```
Comment (miraleung) [inline codecov.yml:3]: Can we make this a non-blocking check?

**Review by miraleung — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (summer-ji-eng) [reply .github/workflows/ci.yaml:63]: Sure

File: codecov.yml (hunk 2 above)
Comment (summer-ji-eng) [reply codecov.yml:3]: Yes. To [set non block status check](https://docs.codecov.io/docs/common-recipe-list#set-non-blocking-status-checks), make the project `informational: true` below.

**Review by miraleung — APPROVED** [review]

**Pull request merged**